governance/votes/Executive Vote - February 4, 2022.md

```diff
@@ -0,0 +1,104 @@
+---
+title: Template - [Executive Vote] Recognized Delegate Compensation, Raise the Emergency Shutdown Threshold, MOMC Parameter Changes - February 02, 2022
+summary: Recognized Delegate Compensation for January 2022, increase the threshold to trigger the Emergency Shutdown Module, Parameter changes from the latest MOMC proposal
```

Comment:
"~~i~~Increase"

Reply:
I think lower case "i" is ok but need to change the capital P to lower case in parameter for consistency - fixed

Reply:
Yeah, that's probably better. I wasn't sure how we were capitalizing these metadata-y list thingies